src/main/java/com/example/event_manager/service/PersonServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package com.example.event_manager.service;
+
+import com.example.event_manager.model.Person;
+
+import java.util.List;
+
+public class PersonServiceImpl implements PersonService {
+  @Override
+  public boolean save(Person personForm) {
+    return false;
+  }
+  
+  @Override
+  public void delete(Long id) {
```

Comment:
Why does it do nothing? If it supposed to do nothing add a comment explaining why